specification/paths/CombinedFiles.json

```diff
@@ -121,20 +144,12 @@
     },
     "responses": {
       "201": {
```

Comment:
For the new RPC endpoint, let's make this `200` like the others that return base64 data.